data/campaigns/Eastern_Invasion/scenarios/03_An_Unexpected_Appearance.cfg

```diff
@@ -1,167 +1,173 @@
+# east is harder: fighting bats/ghosts in difficult terrain, and Mal-Skraat's forces will catch up to you
+#    but leads to the high-reward Bitter Swamp and Mal-Ravanal's Capital
+#
+
+#define TURNS
+24 #enddef
+
 [scenario]
     id=03_An_Unexpected_Appearance
     name= _ "An Unexpected Appearance"
     map_file=03_An_Unexpected_Appearance.map
-    next_scenario=04a_An_Elven_Alliance
-    {TURNS 24 22 20}
+    next_scenario=04a_An_Elven_Interlude
+    turns={TURNS}
 
     {DEFAULT_SCHEDULE}
 
-    {INTRO_AND_SCENARIO_MUSIC breaking_the_chains.ogg vengeful.ogg}
-    {EXTRA_SCENARIO_MUSIC the_king_is_dead.ogg}
-    {EXTRA_SCENARIO_MUSIC wanderer.ogg}
-
+    [story]
+        [part]
+            story=_ "Daylight broke over the relieved faces of Gweddry and his men as they clambered out of the dingy cave. The lush Estmark hills filled the horizon in all directions."
+            music=silence.ogg
+            background=story/S03.jpg
+        [/part]
+    [/story]
     {EI_TRACK {JOURNEY_03_NEW} }
+    {SCENARIO_MUSIC knolls.ogg}
+
+    {PLACE_IMAGE scenery/trapdoor-open.png 23 11}
 
-    # wmllint: validate-off
     [side]
         side=1
         controller=human
-        {GOLD 200 180 160}
+        gold=150
         team_name=wesnothians
         user_team_name= _ "Wesnothians"
         {FLAG_VARIANT loyalist}
-
-        # wmllint: recognize Gweddry
         {CHARACTER_STATS_GWEDDRY}
     [/side]
-    # wmllint: validate-on
 
     [side]
         side=2
         controller=ai
-        recruit=Skeleton,Skeleton Archer,Skeleton Rider,Soulless
-        {GOLD 70 85 100}
-        {INCOME 1 2 3}
+        recruit=Walking Corpse,Soulless,Ghoul,Skeleton Rider
+        {GOLD 20 60 100}
+        {INCOME -2 0 4}
         team_name=undead
         user_team_name= _ "Undead"
         {FLAG_VARIANT undead}
-
         type=Dark Sorcerer
         id=Mal-Skraat
         name= _ "Mal-Skraat"
         canrecruit=yes
-
         facing=ne
         [modifications]
             {TEAM_COLOR_OVERRIDE () darkred}
+            {TRAIT_INTELLIGENT}
+            {TRAIT_STRONG}
         [/modifications]
-
         [ai]
-            recruitment_pattern=scout,scout,fighter,fighter,archer
-        [/ai]
-
-        [ai]
-            recruitment_pattern=fighter,archer,scout,scout
-            aggression=0.6
-            grouping=no
-            [goal]
-                [criteria]
-                    id=Gweddry
-                [/criteria]
-                value=5
-            [/goal]
+            scout_village_targeting=0.8
+            leader_value=0
         [/ai]
     [/side]
+    {STARTING_VILLAGES 2 9}
+    {LIMIT_CONTEMPORANEOUS_RECRUITS 2 "Vampire Bat" 1}
+    {RECRUIT_UNIT_VARIATIONS 2 "Walking Corpse" none,none,mounted,mounted}
+    {RECRUIT_UNIT_VARIATIONS 2 "Soulless" none,none,mounted,mounted}
+    {SILENTLY_LIMIT_LEADER_MOVES 2 1}
 
     [side]
         side=3
         controller=ai
         recruit=Walking Corpse
-        {GOLD 10 20 30}
+        {GOLD 0 0 8}
```

Comment:
`{0 4 8}` or `{0 8 16}` maybe?

Reply:
Note that there's a single initial walking corpse on non-easy difficulties.  So on easy there's no minion, on normal there's 1, and on hard there's 1 plus enough gold to recruit a second.